aggsender/optimistic/optimistic_sign.go

```diff
@@ -76,7 +76,7 @@ func validateSignerAgainstContract(
 	signerAddr common.Address,
 	requireKeyMatch bool,
 ) (common.Address, error) {
-	signers, err := contract.GetAggchainSigners(nil)
+	trustedSequencerAddr, err := getTrustedSequencerAddr(contract)
```

Comment:
Nitpick, consider renaming

```suggestion
	trustedSignerAddr, err := getTrustedSignerAddr(contract)
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/1060/commits/f9b748c5519f66966982027024eb2ab79a880b39